etl/steps/data/garden/oecd/2023-12-06/pisa.yml

```diff
@@ -0,0 +1,12 @@
+# NOTE: To learn more about the fields, hover over their names.
```

Comment:
I think this filename is missing the `metadata` here, so the tag is not being propagated.

Reply:
thanks a lot for spotting this!!